### PR TITLE
Order imports update

### PR DESCRIPTION
**Pull request by rjstires**
## Purpose
Based on lively conversation in Slack I've updated TSLint and all src files to accommodate our import preferences.

Enjoy!

### CODE REVIEW

File: src/features/Volumes/VolumeDrawer.tsx [hunk 1]
```diff
@@ -1,33 +1,14 @@
...
 import { lensPath, path, set, view } from 'ramda';
-
-import { bindActionCreators, compose } from 'redux';
-
+import * as React from 'react';
 import { connect, Dispatch } from 'react-redux';
-
-import { StyleRulesCallback, Theme, withStyles, WithStyles } from '@material-ui/core/styles';
+import { bindActionCreators, compose } from 'redux';
+import * as Rx from 'rxjs/Rx';
```
Comment (martinmckenna) [inline src/features/Volumes/VolumeDrawer.tsx]: should import what we're using

File: src/features/Volumes/Volumes.tsx [hunk 2]
```diff
@@ -1,22 +1,18 @@
...
 import { connect, Dispatch } from 'react-redux';
-import {
-  withStyles,
-  StyleRulesCallback,
-  Theme,
-  WithStyles,
-} from '@material-ui/core/styles';
-import { pathOr, compose } from 'ramda';
+import { bindActionCreators } from 'redux';
+import * as Rx from 'rxjs/Rx';
```
Comment (martinmckenna) [inline src/features/Volumes/Volumes.tsx]: should import what we need

File: src/events.ts [hunk 3]
```diff
@@ -1,23 +1,11 @@
...
-  lensPath,
-  map,
-  not,
-  over,
-  path,
-  view,
-  when,
-} from 'ramda';
+import { assoc, compose, ifElse, isEmpty, isNil, lensPath, map, not, over, path, view, when } from 'ramda';
+import * as Rx from 'rxjs/Rx';
```
Comment (martinmckenna) [inline src/events.ts]: should import what we're using

File: src/features/Volumes/VolumeDrawer.tsx (hunk 1 above)
Comment (rjstires) [reply src/features/Volumes/VolumeDrawer.tsx]: Not really the point of this PR, but not a bad idea either.

**Pull request merged**